rss2email/feed.py

```diff
@@ -654,6 +654,7 @@ def _get_entry_name(self, parsed, entry):
         if 'name' in feed.get('publisher_detail', []):
             data['publisher'] = feed.publisher_detail.name
         name = self.name_format.format(**data)
+        name = name.replace('\n', ' ')
```

Comment:
To make it consistent with this other line:
https://github.com/rss2email/rss2email/blob/9f6ca0af6de81845dd68a1330d45d2345ee422e6/rss2email/feed.py#L594
Add `.strip()` at the end. They both seem to serve the same purpose.